app/models/WhatDoYouWantToDoOptions.scala

```diff
@@ -24,25 +24,46 @@ sealed trait WhatDoYouWantToDoOptions
 object WhatDoYouWantToDoOptions extends Enumerable.Implicits {
 
   case object ArrivalNotifications extends WithName("arrivalNotifications") with WhatDoYouWantToDoOptions
-  case object DepartureDeclarations extends WithName("departureDeclarations") with WhatDoYouWantToDoOptions
+  case object DepartureViewOldDeclarations extends WithName("departureViewOldDeclarations") with WhatDoYouWantToDoOptions
+  case object DepartureMakeDeclarations extends WithName("departureMakeDeclarations") with WhatDoYouWantToDoOptions
+  case object DepartureViewDeclarations extends WithName("departureViewDeclarations") with WhatDoYouWantToDoOptions
   case object NorthernIrelandMovements extends WithName("northernIrelandMovements") with WhatDoYouWantToDoOptions
 
+  val allValues: Seq[WhatDoYouWantToDoOptions] = Seq(
+    ArrivalNotifications,
+    DepartureViewOldDeclarations,
+    DepartureMakeDeclarations,
+    DepartureViewDeclarations,
+    NorthernIrelandMovements
+  )
+
   val values: Seq[WhatDoYouWantToDoOptions] = Seq(
     ArrivalNotifications,
-    DepartureDeclarations,
+    DepartureMakeDeclarations,
     NorthernIrelandMovements
   )
 
-  def radios(form: Form[_]): Seq[Radios.Item] = {
+  def radios(form: Form[_], showDepartures: Boolean): Seq[Radios.Item] = {
```

Comment:
👍

Reply:
Does this require any changes or have we just done extremely well?